pkg/ibmvpc/analysis_output_test.go

```diff
@@ -474,6 +482,13 @@ var tests = []*vpcGeneralTest{
 		grouping:    true,
 		format:      vpcmodel.DRAWIO,
 	},
+	// resource group filtering example
+	{
+		inputConfig:   "multi_resource_groups",
+		useCases:      []vpcmodel.OutputUseCase{vpcmodel.AllEndpoints},
+		format:        vpcmodel.Text,
+		resourceGroup: "ola",
```

Comment:
add in comment what VPCs from the input config file are expected to be filtered out